infrastructure/remediation.tf

```diff
@@ -0,0 +1,151 @@
+data "aws_lambda_function" "ecs_remediator" {
+  function_name = "cyhy-${var.stage}-ecs-remediator"
+}
+
+# --- EventBridge rules + targets for API-gateway 5xx alarms ---
+
+# 1. integration-crossfeed
```

Comment:
Opportunities to condense resources into one.

If there's no difference in the govcloud resources, then we don't need to use `count = var.is_dmz...` and duplicate resources. Instead, one should work with the var.stage variable